_posts/blog/2019-02-28-mlab-announcement.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: blog
+title: "After A Decade of Growth, Measurement Lab Spins Out of New America to Join Code for Science & Society"
+author: "M-Lab Team"
+date: 2019-02-28
+breadcrumb: blog
+categories:
+  - community
+---
+
```

Comment:
Did you want to add the photo here?

Reply:
Yes! I forgot it! Will add in a little bit.